.drone.yml

```diff
@@ -4,7 +4,7 @@ type: kubernetes
 name: test
 
 environment:
-  RUNTIME: go1.13.2
+  RUNTIME: go1.14.4
```

Comment:
Let's not bump the Go version here.
It's not required by any of the 4.3.1 changes and introduces extra risk

Reply:
Good point, I forgot this.